test/external/testcorenrn/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+# Set up the tests from the testcorenrn external repository
+include(NeuronTestHelper)
+set(direct_tests netstimdirect)
+set(spike_comparison_tests
+    bbcore
+    conc
+    deriv
+    gf
+    kin
+    patstim
+    vecplay
+    watch
+    vecevent)
```

Comment:
to partially answer question from slack : these tests can be run either with MPI or serially without MPI. So instead of adding another of set of tests, I wonder if there would be way to say "with mpi enable then run this in parallel with X ranks otherwise run in serial".

Reply:
discussed offline - no ned to worry now.